LayoutTests/imported/w3c/web-platform-tests/webcodecs/image-decoder-utils.js

```diff
@@ -3,7 +3,26 @@ const kRed = 0xFF0000FF;
 const kBlue = 0x0000FFFF;
 const kGreen = 0x00FF00FF;
 
-function getColorName(color) {
+function roundValueAccordingTolerance(value, tolerance)
```

Comment:
Nit: this doesn't really round, maybe `clampValueAccordingTolerance`?

Reply:
Obsolete since I went with another approach